app/controllers/doc_controller.rb

```diff
@@ -65,7 +66,11 @@ def set_doc_version
     return unless @doc_file
     version = params[:version]
     if version
-      @doc_version = @doc_file.doc_versions.for_version(version)
+      if /\d+\.\d+\.\d+/ =~ version
```

Comment:
what's this new logic? :)

Reply:
The idea is that the path in the url can be either <command>/<version> or <command>/<language>. There will be no versioning for translated pages, but the latest master will be published.

Reply:
can we do some very small refactor (or even add an inline comment) so that it's understandable in the future?